pkg/dogstatsd/enrich.go

```diff
@@ -117,15 +117,33 @@ func enrichMetricSample(metricSample dogstatsdMetricSample, namespace string, na
 		hostname = ""
 	}
 
-	return metrics.MetricSample{
+	mtype := enrichMetricType(ddSample.metricType)
+
```

Comment:
Again, it's nit but while reading, the difference between `value` and `values` is small: could we comment here something like
```
// contains multiple values, unroll them and return them
```
and below
```
// only one value contained, simple append it
```

or even mentioning the protocol version. WDYT?

Reply:
I added two comments, let me know what you think